sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/FileSample.java

```diff
@@ -90,8 +88,8 @@ public static void main(String[] args) throws Exception {
         }
 
         // Upload a local file to the storage.
-        URL fileFolder = FileSample.class.getClassLoader().getResource(".");
-        String uploadPath = fileFolder.getPath() + "testfiles/" + "uploadSample.txt";
+        String filePath = "C:/filePath/";
```

Comment:
What's the reason change to C folder?

Reply:
Using the class loader get resource could lead to a NPE, given this is a sample not tying to anything real I just made it simpler.